WORKSPACE

```diff
@@ -106,6 +90,24 @@ load("@io_grpc_grpc_java//:repositories.bzl", "IO_GRPC_GRPC_JAVA_ARTIFACTS", "IO
 
 grpc_java_repositories()
 
+# Google Protocol Buffers.
+# https://github.com/protocolbuffers/protobuf
+#
+# Protobuf should be loaded after `grpc_deps` in the WORKSPACE file
```

Comment:
I think this refers to `protobuf_deps` below, right? In which case it should probably go there. Or does it refer to this http_archive?

Reply:
Done